gradle/libs.versions.toml

```diff
@@ -0,0 +1,43 @@
+## Note:
+##
+## Add dependencies that you want to use in this definition.
+## You can access these after a gradle sync via:
+##
+## dependencies {
+##     implementation(libs.vertx.web)
+## }
```

Comment:
I believe you, but am curious how this works. Given that it's notated as `vertx-web` below (and the package name also is `vertx-web`), how does it infer that from the `.` in `vertx.web`?

Reply:
Gradle automatically converts the separators to the `.` separator used in this example.

> When declaring aliases, it’s worth noting that any of the -, _ and . characters can be used as separators, but the generated catalog will have all normalized to .: for example foo-bar as an alias is converted to foo.bar automatically.
> https://docs.gradle.org/current/userguide/platforms.html